### PR TITLE
mxs: run kernel_oldconfig

### PR DESCRIPTION
**Pull request by neheb**

### CODE REVIEW

File: target/linux/mxs/config-6.6 [hunk 1]
```diff
@@ -86,13 +88,14 @@ CONFIG_EXTCON=y
...
 CONFIG_FIX_EARLYCON_MEM=y
+CONFIG_FORCE_NR_CPUS=y
 CONFIG_FS_IOMAP=y
 CONFIG_FS_MBCACHE=y
 CONFIG_FUNCTION_ALIGNMENT=0
 CONFIG_FWNODE_MDIO=y
 CONFIG_FW_LOADER_PAGED_BUF=y
 CONFIG_FW_LOADER_SYSFS=y
-CONFIG_GCC11_NO_ARRAY_BOUNDS=y
+CONFIG_GCC10_NO_ARRAY_BOUNDS=y
```
Comment (Ansuel) [inline target/linux/mxs/config-6.6:98]: mhhh really sounds like a good candiate for the filter...

Comment (neheb) [reply target/linux/mxs/config-6.6:98]: Yep. https://github.com/openwrt/openwrt/pull/16770

Comment (neheb) [reply target/linux/mxs/config-6.6]: removed.

**Comment by neheb** [comment]
ping @robimarko

**Pull request merged**